python/lsst/afw/fits/SConscript

```diff
@@ -1,3 +1,3 @@
 ## -*- python -*-
 #from lsst.sconsUtils import scripts
-#scripts.BasicSConscript.python(['fitsLib'])
+#scripts.BasicSConscript.pybind11(['fits'])
```

Comment:
I'm going to push back on this just because this is a template file that is automatically created in that directory. I do agree with the other places that you are requesting more information but there are SConscript files commented out exactly like this all over afw and are only uncommented when we finally wrap them. The only reason this one showed up is because I started to wrap this directory until I realized that it was unnecessary.